app/src/main/java/chat/rocket/android/fragment/server_config/LoginFragment.kt

```diff
@@ -48,6 +54,38 @@ class LoginFragment : AbstractServerConfigFragment(), LoginContract.View {
         val btnUserRegistration = rootView.findViewById<Button>(R.id.btn_user_registration)
         txtUsername = rootView.findViewById(R.id.editor_username)
         txtPasswd = rootView.findViewById(R.id.editor_passwd)
+        textInputUsername = rootView.findViewById(R.id.text_input_username)
+        textInputPassword = rootView.findViewById(R.id.text_input_passwd)
+
+        //setting text change listeners to username and password edit texts
+        txtUsername.addTextChangedListener(object :TextWatcher
+        {
+            override fun afterTextChanged(p0: Editable?) {
+            }
+
+            override fun beforeTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {
+            }
+
+            override fun onTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {
+                if(!TextUtils.isEmpty(txtUsername.text.toString()) && textInputUsername.isErrorEnabled)
+                    textInputUsername.setErrorEnabled(false)
+            }
+        })
+
+        txtPasswd.addTextChangedListener(object :TextWatcher
+        {
+            override fun afterTextChanged(p0: Editable?) {
+            }
+
+            override fun beforeTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {
+            }
+
+            override fun onTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {
+                if(!TextUtils.isEmpty(txtPasswd.text.toString()) && textInputPassword.isErrorEnabled)
+                    textInputPassword.setErrorEnabled(false)
+            }
+        })
+        
```

Comment:
I know the IDE names those parameters p0, p1, p2, and son on automatically but when we override some method or anonymous classes, I think we should avoid these defaults and properly naming each one of the parameters.